### DISCUSSION TITLE
Compact mod menu option

### DISCUSSION THREAD

**Discussion by Doksha** [ppy/osu#20077] [Ideas]
With more and more categories being added to the mods menu (recently presets, and now the proposed "Restriction" category), its getting more and more difficult to activate mods (especially difficulty increase mods). An option in settings to change the mods into a more compact view, with a more similar layout to stable (preset icons on top, diff reduction 2nd row, restriction mods 3rd row, difficulty increase 3rd row, auto mods 4th row, and fun mods 5th and 6th rows), where you have just icons, and hovering over them shows you the name and description of the mod.
Obviously the current system would be default, but for anyone who has used lazer for a while and knows the icons, this would be a nice change to the system.

**Comment by nagi-desuuu** [comment]
They once had this layout up until 3-6 months ago; the reason they changed it (I think) was because of design reasons. Correct me if I'm wrong.

**Reply by Doksha** [reply]
would still be nice as an option. The current layout is fine for new players, but it gets tedious after a while
